index.html

```diff
@@ -108,5 +108,83 @@ <h4>Skills</h4>
+        <div class="work-tittle">
+          <h1>MY RECENT <br><br> WORK</h1>
+          <hr>
+        </div>
+        <div class="projects">
+          <div class="projects-seection">
+            <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+            <h2 class="post-tittle">Multi-Post Stories</h2>
+            <div class="support-text">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.<br>
+                has been the industry's standard dummy text ever since the 1500s,<br>
+                when an unknown printer took a standard dummy text.
+              </p>
+              <ul>
+                <li> css </li>
+                <li> html</li>
+                <li>bootstrap</li>
+                <li>Ruby </li>
+              </ul>
+              <button class="bttn" type="button">See Project</button>
+            </div>
+          </div>
+          <div class="projects-seection">
+            <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+            <h2 class="post-tittle">Multi-Post Stories</h2>
+            <div class="support-text">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.<br>
+                has been the industry's standard dummy text ever since the 1500s,<br>
+                when an unknown printer took a standard dummy text.
+              </p>
+              <ul >
+                <li> css </li>
+                <li> html</li>
+                <li>bootstrap</li>
+                <li>Ruby </li>
+              </ul>
+              <button class="bttn" type="button">See Project</button>
+          </div>
+          </div>
+          <div class="projects-seection">
+            <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+            <h2 class="post-tittle">Multi-Post Stories</h2>
+            <div class="support-text">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.<br>
+                has been the industry's standard dummy text ever since the 1500s,<br>
+                when an unknown printer took a standard dummy text.
+              </p>
+                <ul>
+                   <li> css </li>
+                  <li> html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby </li>
+                </ul>
+                <button class="bttn" type="button">See Project</button>
+            </div>
+          </div>
+          <div class="projects-seection">
+            <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+            <h2 class="post-tittle">Multi-Post Stories</h2>
+            <div class="support-text">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.<br>
+                has been the industry's standard dummy text ever since the 1500s,<br>
+                when an unknown printer took a standard dummy text.
```

Comment:
- [ Optional ] You should use a `font-family` of Inter and `font-size` of 20px for the project description text.